### PR TITLE
Update docker version.

### PR DESCRIPTION
**Pull request by metanerd**
#### Summary
Updates docker version.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -19,8 +19,8 @@ ENV MYSQL_DATABASE=mattermost_test
 WORKDIR /mm
 
 # Copy over files
-ADD https://releases.mattermost.com/5.24.2/mattermost-team-5.24.2-linux-amd64.tar.gz .
-RUN tar -zxvf ./mattermost-team-5.24.2-linux-amd64.tar.gz
+ADD https://releases.mattermost.com/5.26.0/mattermost-team-5.26.0-linux-amd64.tar.gz .
+RUN tar -zxvf ./mattermost-team-5.26.0-linux-amd64.tar.gz
```
Comment (cpanato) [inline Dockerfile:23]: use 5.26.1

**Review by cpanato — CHANGES_REQUESTED** [review]
small change

File: Dockerfile [hunk 2]
```diff
@@ -19,8 +19,8 @@ ENV MYSQL_DATABASE=mattermost_test
 WORKDIR /mm
 
 # Copy over files
-ADD https://releases.mattermost.com/5.24.2/mattermost-team-5.24.2-linux-amd64.tar.gz .
-RUN tar -zxvf ./mattermost-team-5.24.2-linux-amd64.tar.gz
+ADD https://releases.mattermost.com/5.26.0/mattermost-team-5.26.0-linux-amd64.tar.gz .
```
Comment (metanerd) [inline Dockerfile:22]:
```suggestion
ADD https://releases.mattermost.com/5.26.1/mattermost-team-5.26.1-linux-amd64.tar.gz .
```

File: Dockerfile (hunk 1 above)
Comment (metanerd) [inline Dockerfile:23]:
```suggestion
RUN tar -zxvf ./mattermost-team-5.26.1-linux-amd64.tar.gz
```

**Review by cpanato — APPROVED** [review]

**Pull request merged**